_data/simple-icons.json

```diff
@@ -270,6 +310,11 @@
             "hex": "EE6123",
             "source": "https://bitly.com/pages/press"
         },
+        {
+            "title": "Blackberry",
+            "hex": "00BFFF",
```

Comment:
I think this should be black `#000000`. If we choose blue, we should use "BlackBerry Blue" `#005387`, but they don't seem to use that much.